core/jobs_test.py

```diff
@@ -799,15 +956,9 @@ def _increment_counter():
             realtime_model_id = realtime_class.get_realtime_id(
                 active_realtime_layer, exp_id)
 
-            realtime_model = realtime_class.get(
```

Comment:
Why have you changed this code? I see no explanation.

Reply:
These lines of code were not used in our tests